gemd/util/tests/test_cached_isinstance.py

```diff
@@ -0,0 +1,30 @@
+from gemd.util import cached_isinstance
+
+from typing import List, Iterable
+
+
+def test_cached_isinstance():
```

Comment:
Did we not already have a test like this?

Reply:
We did not because it was never publicly exposed.